app/(pages)/event/page.tsx

```diff
@@ -99,11 +82,40 @@ export default async function Page({
 
   return (
     <AppContainer user={user}>
-      <EventList
-        events={events}
-        pageMeta={pageMeta}
-        validPagination={validPagination}
-      />
+      <div className="bg-zinc-900">
+        <div className="min-viewport-height mx-auto flex h-full w-full max-w-7xl flex-1 flex-col  px-4">
+          <Header logoText="inLive Room" logoHref="/" />
+          {TrialMessage()}
+          <EventList
+            events={events}
+            pageMeta={pageMeta}
+            validPagination={validPagination}
+            limitCreate={isLimitReached}
+          />
+        </div>
+      </div>
```

Comment:
Kenapa enggak dimasukkan ke component EventList sekalian aja gah? Kenapa harus dikeluarin? Kalo function return jsx itu bisa dipanggil kayak component aja `<TrialMessage />`.

Reply:
alasannya untuk simplicity aja, berhubung page ini juga memang hanya menampilkan event list, dan TrialMessage ini akan di remove setelah beres dari limited beta

Komponent TrialMessage ini juga tidak dependent dengan prop EventList seperti events, pagemeta, dan validPagination menurutku better dipisah saja untuk minimalisir prop passing yang

mungkin untuk selanjutnya juga better komponent seperti header dan footer ini dipindahkan ke layout saja, daripada sekarang yang dipanggil berulang ulang disetiap komponent

https://nextjs.org/docs/pages/building-your-application/routing/pages-and-layouts#layout-pattern